src/celpy/celtypes.py

```diff
@@ -442,6 +442,9 @@ def __new__(
     def __repr__(self) -> str:
         return f"{self.__class__.__name__}({super().__repr__()})"
 
+    def contains(self, item: Value) -> BoolType:
+        return BoolType(cast(StringType, item) in self)
```

Comment:
This isn't passing tests (for me, or in CI) Perhaps we should do this instead?
```suggestion
        return BoolType(BytesType(item) in self)
```

Reply:
Or maybe you meant for this to be `cast(BytesType, item)` but if so you'd want to do assert `b_0.contains(b"byte")` below...

Reply:
Okay, did a bit more testing, and it seems like this is what's necessary to appease the type checker (and what's consistent with other examples of `contains()`):

```suggestion
        return BoolType(cast(bytes, item) in cast(bytes, self))
```